recipes/medicc2/meta.yaml

```diff
@@ -23,15 +23,15 @@ requirements:
     - python
     - pip
     - numpy >=1.20.1
-    - openfst >=1.8.1
+    - openfst ==1.8.1
```

Comment:
The `host` section only needs to contain those dependencies required at `pip install` time. Since your `setup.py` doesn’t import numpy nor openfst, you can delete them from here.

Reply:
I tried that in a previous version and the builds failed since MEDICC2 extends the `openfst` packages as a `cython` extension. This step also requires `numpy`. 

I added a comment to prevent confusion about this in the future.

Reply:
Makes sense!